helper/schema/schema.go

```diff
@@ -201,6 +201,23 @@ func (s *Schema) GoString() string {
 	return fmt.Sprintf("*%#v", *s)
 }
 
+// Returns a default value for this schema by either reading Default or
+// evaluating DefaultFunc. If neither of these are defined, returns nil.
+func (s *Schema) GetDefaultValue() (interface{}, error) {
+	// Easier to just preload Default even if it's nil than to do another
+	// conditional here
+	defaultValue := s.Default
```

Comment:
If `s.Default` is not set (we can check `s.Default == nil` since it is an `interface{}` type), then we should probably use `s.Zero()`, which is another method I made to get a zero value.